recipes/ogs5py/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ogs5py" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: dfde1763dd033ebeef380d098ffb8af47e8f72dead477bb6c9eeceeb342f9358
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - lxml
+    - meshio
```

Comment:
You will need to create a package for ``meshio`` , there is no such package on ``conda-forge``  :/

Reply:
Just did that: https://github.com/conda-forge/staged-recipes/pull/10194